.github/workflows/build.yml

```diff
@@ -91,8 +91,8 @@ jobs:
           path: test/bats-core
       - run: test/bats-core/install.sh $HOME/.local
 
-      - name: Install git-pile
-        run: python3 -m pip install --user .
+      - name: Use git-pile from checkout
+        run: echo "$PWD" >> $GITHUB_PATH
```

Comment:
Why do we need to use it directly from the checkout? Could we use editable install instead?

The main reason I'm asking is because in the later commit adding coverage support ("github: Output coverage results") we can use, for example, `pip install -e .[tests]`. In fact, I even believe that that command should be done here, since the `[tests]` would install any extra requirement for testing: currently only the `coverage` package, but there could be additional ones in the future.

Reply:
Okay.